documentation/md/style.md

```diff
@@ -297,6 +297,7 @@ A background image may be applied to a node's body.  The following properties su
     * The [`cytoscape-sbgn-stylesheet`](https://github.com/PathwayCommons/cytoscape-sbgn-stylesheet) package serves as a good example for the use of SVG images in a stylesheet.  That stylesheet [creates decorations](https://pathwaycommons.github.io/cytoscape-sbgn-stylesheet/) on nodes in line with the [SBGN standard](https://sbgn.github.io).
 * **`background-image-crossorigin`**: All images are loaded with a [`crossorigin`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-crossorigin) attribute which may be `anonymous` or `use-credentials`. The default is set to `anonymous`.
 * **`background-image-opacity`** : The opacity of the background image.
+* **`background-image-smoothing`** : Determines whether background image is smoothed (`yes`, default) or not (`no`). This feature isn't well supported across browsers.
```

Comment:
Could this be made a bit more explicit that it's a only a hint?  The browser may or may not respect the value set for this property.